src/object.d

```diff
@@ -3634,6 +3634,12 @@ if (!__traits(isScalar, T1))
     assert(__cmp([c2, c2], [c1, c1]) > 0);
 }
 
+template _arrayOp(Args...)
```

Comment:
The purpose of this is mysterious. Is it to just forward to `core.internal.arrayop.arrayOp` ? If so, why not just use alias? Please add documentation comment.